### PR TITLE
Disable all {problems} text when settings is disabled

### LINKED ISSUE #127: Disable Problems status is broken
Hello.
When disabling the `Problems` status in `Vscord > Status`,
it does disable the `problems_count`, but it doesn't disable the rest of the text

Version: 5.0.4

### PR DESCRIPTION
**Pull request by nick22985**
Fixes #127 prob a better way of doing this

### CODE REVIEW

**Review by leonardssh — APPROVED** [review]

**Comment by xhayper** [comment]
Just saw this, my bad!

File: src/activity.ts [hunk 1]
```diff
@@ -65,12 +65,12 @@ export const activity = async (
...
     isViewing = !isDebugging && isViewing;
 
-    const PROBLEMS = await replaceFileInfo(
+    const PROBLEMS =  config.get(CONFIG_KEYS.Status.Problems.Enabled) ? await replaceFileInfo(
         replaceGitInfo(replaceAppInfo(config.get(CONFIG_KEYS.Status.Problems.Text)), isGitExcluded),
         isWorkspaceExcluded,
         dataClass.editor?.document,
         dataClass.editor?.selection
-    );
+    ) : '';
```
Comment (xhayper) [inline src/activity.ts:73]: use `FAKE_EMPTY` instead of `''`

**Review by xhayper — CHANGES_REQUESTED** [review]

**Review by xhayper — COMMENTED** [review]
LGTM!

**Comment by xhayper** [comment]
idk how that comment get there, ignore it

**Pull request merged**